scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -99,9 +99,21 @@ The Callset Stats and S&P files can be simply `gsutil cp`ed to the AoU delivery
 ## Running the VAT pipeline
 To create a BigQuery table of variant annotations, you may follow the instructions here:
 [process to create variant annotations table](../../variant_annotations_table/README.md)
-
 The pipeline takes in the VDS and outputs a variant annotations table in BigQuery.
 
+Once the VAT table and a tsv is exported, the AoU research workbench team should be notified of its creation and permission should be granted so that several members of the team have view permission.
+
+- Grant `BigQuery Data Viewer` permission to specific people's PMI-OPS accounts. As of the Echo release, these accounts were:
+        <ul>
+            <li>kevin.levy@pmi-ops.org</li>
+            <li>srushti.gangireddy@pmi-ops.org</li>
+            <li>brian.freeman@pmi-ops.org </li>
+            <li>tarek.ahmed@pmi-ops.org</li>
+            <li>jennifer.zhang@pmi-ops.org</li>
```

Comment:
do we really want to put these emails in our docs?

Reply:
Yeah - probably shouldn't but wasn't sure how to capture this level of detail.

Reply:
maybe link back to a Confluence or something?

Reply:
I've linked to a Google Doc that tracks what we did for Echo